include/modem/lte_lc.h

```diff
@@ -471,6 +472,22 @@ enum lte_lc_evt_type {
 	 */
 	LTE_LC_EVT_ENV_EVAL_RESULT		= 13,
 #endif /* CONFIG_LTE_LC_ENV_EVAL_MODULE */
+
+	/**
+	 * PDN event
+	 *
+	 * The associated payload is the @c lte_lc_evt.pdn member of type
+	 * @ref lte_lc_pdn_evt in the event.
+	 */
+	LTE_LC_EVT_PDN				= 14,
+
+	/**
+	 * A cellular profile is activated for the current access technology.
+	 *
+	 * The associated payload is the @c lte_lc_evt.cellular_profile member of type
+	 * @ref lte_lc_cellular_profile_evt in the event.
+	 */
+	LTE_LC_EVT_CELLULAR_PROFILE_ACTIVE	= 15,
```

Comment:
These and `LTE_LC_EVT_NW_REG_STATUS` should be flagged with new corresponding Kconfigs.